posts/how-to-commit-code-to-a-git-repository-with-visual-studio-code.md

```diff
@@ -0,0 +1,172 @@
+
+## Step 1 — Opening the Source Control panel
+
+The **Source Control** panel in VS Code allows you to use Git features. It’s one of the panels that you can select in the **Activity Bar**, which is the vertical bar on the left side of the VS Code interface.
+
+Each panel in the **Activity Bar** is represented by an icon. The **Source Control** icon is highlighted in the screenshot below:
+
+<figure className="small">
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-icon.png" alt="Location of the Source Control icon in the Activity Bar." />
+    <figcaption>Location of the <strong>Source Control</strong> icon in the Activity Bar.</figcaption>
+</figure>
+
+Click on the icon to open the **Source Control** panel. This panel provides an interface to view the current changes in your working directory and to create new commits.
+
+<figure className="small">
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-source-control-panel.png" alt="The Source Control panel displays some changes in the repository." />
+    <figcaption>The <strong>Source Control</strong> panel displays some changes in the repository.</figcaption>
+</figure>
+
+The **Changes** section of the panel lists all the files in the working directory that have been changed. The total number of changed files is also reflected on the **Source Control** icon’s badge. In the example in the above screenshot, the badge indicates that there are three changed files, and those three files are listed in the **Changes** section.
+
+By selecting one of the files in the list, we can view the changes to that file. VS Code displays the changes as a **diff view** in a new editor tab. For example, if we select the `README.md` file, we can browse the changes as displayed in the screenshot below.
+
+<figure>
+    <img src="https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/vs-code-diff-view.png" alt="An example of a diff view of the changes made to README.md." />
+    <figcaption>An example of a diff view of the changes made to <code>README.md</code>.</figcaption>
+</figure>
+
+To prepare a commit, we will select changes to add to the staging area.
+
+## Step 2 — Adding changes to the staging area
+
+The **staging area** in Git is the collection of changes that will be included in the next commit. The next step is to add some changes to the staging area.
+
+### Staging individual files
+
+Each file in the **Changes** list represents a change in the repository’s working directory that has not yet been staged and is available for staging.
+
+The badge next to each file indicates the type of change:
+
+- The `M` badge indicates that a file is modified. This means that it exists in the repository and has changed since the last commit.
```

Comment:
I think it could be helpful to bold the letters that correspond to the badges, like:

```
- The `M` badge indicates that a file is **m**odified. This means that it exists in the repository and has changed since the last commit.
- The `U` badge represents an **u**ntracked file, which means it doesn’t exist in the repository and was newly created since the last commit.
- The `D` badge means that the file was **d**eleted.
```